### PR TITLE
Cran release fixes

### PR DESCRIPTION
**Pull request by krystian8207**
Created new LICENSE.md file with full MIT license text. LICENSE file shortened as MIT template requires.
Fixed other simple notes such as mis-spelled words and form of urls in Description section.

### CODE REVIEW

File: DESCRIPTION [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
-Title: Semantic Ui Support for Shiny
+Title: Semantic UI Support for Shiny
 Version: 0.1.1
 Authors@R: person("Filip", "Stachura", email = "filip@appsilondatascience.com",
   role = c("aut", "cre"))
 Description: Creating great user interface for your Shiny apps
-    can be a hassle, especialy if you want to work purely in R
+    can be a hassle, especially if you want to work purely in R
     and don't want to use, for instance HTML templates. This
     package add support for powerful UI library Semantic UI -
```
Comment (KrzysztofWrobel) [inline DESCRIPTION]: This package adds support for a powerful UI library Semantic UI

File: DESCRIPTION [hunk 2]
```diff
@@ -1,14 +1,14 @@
 Package: shiny.semantic
 Type: Package
-Title: Semantic Ui Support for Shiny
+Title: Semantic UI Support for Shiny
 Version: 0.1.1
 Authors@R: person("Filip", "Stachura", email = "filip@appsilondatascience.com",
   role = c("aut", "cre"))
 Description: Creating great user interface for your Shiny apps
```
Comment (KrzysztofWrobel) [inline DESCRIPTION]: ... a great ...

File: DESCRIPTION [hunk 3]
```diff
@@ -1,14 +1,14 @@
...
-    can be a hassle, especialy if you want to work purely in R
+    can be a hassle, especially if you want to work purely in R
     and don't want to use, for instance HTML templates. This
     package add support for powerful UI library Semantic UI -
-    http://semantic-ui.com/. You can also find support for
+    <http://semantic-ui.com/>. You can also find support for
```
Comment (KrzysztofWrobel) [inline DESCRIPTION]: Change: You can also find support for -> It also supports universal ...

**Comment by krystian8207** [comment]
Changes made, as requested.

**Pull request merged**